pkg/csilvm/server.go

```diff
@@ -87,10 +81,9 @@ func SupportedFilesystem(fstype string) ServerOpt {
 	}
 }
 
-// RemoveVolumeGroup configures the Server to operate in "remove"
-// mode. The volume group will be removed when NodeProbe is
-// called. All RPCs other than GetSupportedVersions, GetPluginInfo and
-// NodeProbe will fail if the plugin is started in this mode.
+// RemoveVolumeGroup configures the Server to operate in "remove" mode. The
+// volume group will be removed when Probe is called. All RPCs other than those
```

Comment:
"All other RPCs ..." -- this isn't strictly true. for example ListVolumes is still needed. As is GetCapacity